tm2/pkg/sdk/baseapp.go

```diff
@@ -653,6 +653,7 @@ func (app *BaseApp) runMsgs(ctx Context, msgs []Msg, mode RunTxMode) (result Res
 		// each result.
 		data = append(data, msgResult.Data...)
 		events = append(events, msgResult.Events...)
+		events = append(events, ctx.eventLogger.Events()...)
```

Comment:
Why would we need to append the events from the `eventLogger`?

Reply:
removed. I thought it might be use the `Context` type later. But it seems unnecessary even later.